.gitignore

```diff
@@ -1,3 +1,9 @@
+#emacs
+*~
+
+# JetBrains IDE stuff
+.idea
+
```

Comment:
Would you please remove the change of `.gitignore` as it's not part of the issue we're going to solve here? Thanks.